src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -22,10 +24,24 @@ export type Props = PlainButtonProps & {
 
 export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
 
-export function SidebarNavSign({ intl, status, targetingApi, ...rest }: Props) {
+export function SidebarNavSign({ blockedReason, intl, isDisabled, status, targetingApi, ...rest }: Props) {
     const isTargeted = targetingApi && targetingApi.canShow;
     const FtuxTooltip = isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
```

Comment:
Should we disable FTUX if the file is blocked? Otherwise it could lead to a poor experience.

Reply:
yeah that makes sense